config.go

```diff
@@ -176,12 +176,12 @@ type ticketBuyerOptions struct {
 	MinFee             *cfgutil.AmountFlag `long:"minfee" description:"Minimum ticket fee per KB"`
 	FeeSource          string              `long:"feesource" description:"The fee source to use for ticket fee per KB (median or mean)"`
 	MaxPerBlock        int                 `long:"maxperblock" description:"Maximum tickets per block, with negative numbers indicating buy one ticket every 1-in-n blocks"`
-	HighPricePenalty   float64             `long:"highpricepenalty" description:"The exponential penalty to apply to the number of tickets to purchase above the ideal ticket pool price"`
 	BlocksToAvg        int                 `long:"blockstoavg" description:"Number of blocks to average for fees calculation"`
 	FeeTargetScaling   float64             `long:"feetargetscaling" description:"The amount above the mean fee in the previous blocks to purchase tickets with, proportional e.g. 1.05 = 105%"`
 	DontWaitForTickets bool                `long:"dontwaitfortickets" description:"Don't wait until your last round of tickets have entered the blockchain to attempt to purchase more"`
 	MaxInMempool       int                 `long:"maxinmempool" description:"The maximum number of your tickets allowed in mempool before purchasing more tickets"`
 	ExpiryDelta        int                 `long:"expirydelta" description:"Number of blocks in the future before the ticket expires"`
+	MaxPriceRelative   float64             `long:"maxpricerelative" description:"The max price to pay for tickets relative to the average price"`
```

Comment:
the description needs to be changed to make it clear that it is a scaling factor then.  currently it sounds like an amount that is added/subtracted from some other amount to determine the final value